ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -180,16 +413,62 @@ const triggerSet: TriggerSet<Data> = {
       netRegexDe: NetRegexes.startsUsing({ id: '69D6', source: 'Hesperos', capture: false }),
       netRegexFr: NetRegexes.startsUsing({ id: '69D6', source: 'Hespéros', capture: false }),
       netRegexJa: NetRegexes.startsUsing({ id: '69D6', source: 'ヘスペロス', capture: false }),
-      infoText: (_data, _matches, output) => output.text!(),
+      preRun: (data) => data.pinaxCount = (data.pinaxCount ?? 0) + 1,
+      infoText: (data, _matches, output) => {
+        if ((data.pinaxCount ?? 0) % 2)
+          return output.text!();
+        data.wellShiftKnockback = true;
+        return output.shiftWell!();
+      },
       outputStrings: {
         text: {
           en: 'Well Pinax',
           de: 'Brunnen-Pinax',
           fr: 'Pinax d\'eau',
+          ja: '水ピナクス',
           cn: '水',
+          ko: '물 피낙스',
+        },
+        shiftWell: {
+          en: 'Well then Shift',
```

Comment:
small suggestion but, "Well + Shift" or "Well => Shift" for more consistency with other cactbot texts?

Reply:
I think this makes sense, wasn't too sure on how to output it.